timelock-agent/src/main/java/com/palantir/timelock/config/TimeLockRuntimeConfiguration.java

```diff
@@ -39,6 +40,12 @@ public PaxosRuntimeConfiguration paxos() {
         return ImmutablePaxosRuntimeConfiguration.builder().build();
     }
 
+    /**
+     *  The token required by other services to use backup and restore-related endpoints.
+     */
+    @JsonProperty("permitted-backup-token")
+    public abstract BearerToken permittedBackupToken();
+
```

Comment:
Does it have to be a required field?

Reply:
I think it should be. Adding it will be zero lift for consumers (we would do it automatically in Timelock), and I don't see a situation where we wouldn't want to protect the usage of these endpoints.

Reply:
By the way, I checked and it looks like adding this as a required field will not cause any unexpected breaks in products that use AtlasDB.